powersimdata/data_access/data_access.py

```diff
@@ -1,36 +1,33 @@
-import glob
 import operator
-import os
 import posixpath
-import shutil
 import time
 from subprocess import Popen
-from tempfile import mkstemp
 
-import paramiko
-from tqdm import tqdm
+import fs as fs2
+from fs.tempfs import TempFS
 
 from powersimdata.data_access.profile_helper import ProfileHelper
+from powersimdata.data_access.ssh_fs import WrapSSHFS
 from powersimdata.utility import server_setup
-from powersimdata.utility.helpers import CommandBuilder
 
-_dirs = {
-    "tmp": (server_setup.EXECUTE_DIR,),
-    "input": server_setup.INPUT_DIR,
-    "output": server_setup.OUTPUT_DIR,
-}
+
+def get_ssh_fs(root=None):
+    if root is None:
+        root = ""
+    host = server_setup.SERVER_ADDRESS
+    port = server_setup.SERVER_SSH_PORT
+    username = server_setup.get_server_user()
+    base_fs = fs2.open_fs(f"ssh://{username}@{host}:{port}")
+    return WrapSSHFS(base_fs, root)
 
```

Comment:
Is there any reason this function is not a method of the `SSHDataAccess` class?

Reply:
No reason. I found it convenient to be able to use interactively without creating an SSHDataAccess instance though